packages/@okta/vuepress-site/docs/reference/api/log-streaming/index.md

```diff
@@ -685,7 +685,7 @@ The Splunk Cloud Settings object specifies the configuration for the `splunk_clo
 
 ```json
 {
-  "host": "acme.splunkcloud.com"
+  "host": "acme.splunkcloud.com",
   "token": "11111111-1111-2222-2222-222222222222"
```

Comment:
```suggestion
  "token": "11111-redacted"
```
Is the format of the returned token `#####-redacted` or just hashed-out, like `********` ?

Reply:
Upon discussion on the API shape, the token field will not be returned from the API at all

Reply:
Ok, we can leave it as-is for a creation example.